lib/services/models.js

```diff
@@ -111,14 +112,14 @@ function get(model, renderModel, executeRender, uri, locals) { /* eslint max-par
       }
 
       if (ms > timeoutLimit * 0.5) {
-        let slowMsg = ''
-          + `slow get ${uri} ${ms}ms`
-          + `\n  db.get:  ${times.afterGet - times.start}ms`
-          + `\n  upgrade: ${times.afterUpgrade - times.afterGet}ms`
-          + `\n  render:  ${times.afterRender - times.afterUpgrade}ms`;
+        const dbDuration = times.afterGet - times.start,
+          upgradeDuration = times.afterUpgrade - times.afterGet,
+          renderDuration = times.afterRender - times.afterUpgrade;
+        let detailedRenderTimes = '';
+
 
         if (locals.amphoraRenderTimes.length) {
-          const timesDuringRender = locals.amphoraRenderTimes
+          detailedRenderTimes = locals.amphoraRenderTimes
             .map(({ data, label, ms }) => {
               let msg = `  ${label}: ${ms}ms`;
```

Comment:
These should be kept in array format and drop the `ms` so that we can properly parse it in DD